test/clojure-mode-font-lock-test.el

```diff
@@ -101,7 +101,13 @@ POS."
 
 (ert-deftest clojure-mode-syntax-table/fontify-clojure-keyword ()
   :tags '(fontification syntax-table)
-  (should (eq (clojure-test-face-at 2 11 "{:something 20}") 'clojure-keyword-face)))
+  (should (equal (clojure-test-face-at 2 11 "{:something 20}") '(clojure-keyword-face))))
+
+(ert-deftest clojure-mode-syntax-table/fontify-namespaced-keyword ()
+  :tags '(fontification syntax-table)
+  (should (equal (clojure-test-face-at 2 2 "{:alias/some 20}")  '(clojure-keyword-face)))
```

Comment:
Guys, why? It's very inconvenient now and it's weird.

Namespace within keyword should always be highlighted with the same color as keyword are, as it always was here and in all other editors, including vim and intellij.

It's so inconvenient that I'm looking now for a hack, how can I revert this behavior back without changing clojure-mode version...